### PR TITLE
[snapshot] Use 7.11.0-SNAPSHOT as testing stack

### PR DESCRIPTION
**Pull request by mtojek**
This PR updates the Elastic testing stack to 7.11.0.

### CODE REVIEW

**Comment by mtojek** [comment]
jenkins run the tests please

**Comment by mtojek** [comment]
jenkins run the tests please

**Comment by mtojek** [comment]
@ruflin I'm trying to update the package storage to work with Kibana 7.11.0, but it fails in a weird way. Do you have any clues where can be a difference between both revisions?

**Comment by mtojek** [comment]
jenkins run the tests please

**Comment by mtojek** [comment]
Ok, fixed it. It looks like in 7.11.0 package are installed slower than in 7.10.0

File: magefile.go [hunk 1]
```diff
@@ -177,5 +177,5 @@ func TestIntegration() error {
 	if err != nil {
 		return err
 	}
-	return sh.RunV("go", "test", "testing/main_integration_test.go", "-v", "-tags=integration", "2>&1", "|", "go-junit-report", ">", "junit-report.xml")
+	return sh.RunV("go", "test", "testing/main_integration_test.go", "-v", "-tags=integration", "-test.timeout", "30m", "2>&1", "|", "go-junit-report", ">", "junit-report.xml")
```
Comment (ycombinator) [inline magefile.go:180]: 👍

**Review by ycombinator — APPROVED** [review]
LGTM.

Ironic that a PR whose number is #710 is the one replacing version 7.10.0 🙂

**Pull request merged**